contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -225,6 +227,10 @@ func NewSchedulerServer() *SchedulerServer {
 		// when kube-proxy is running in a non-root netns (init_net); setting this to a non-zero value will
 		// impact connection tracking for the entire host on which kube-proxy is running. xref (k8s#19182)
 		conntrackMax: 0,
+
+		// cadvisor housekeeping related config
+		housekeepingInterval:       1 * time.Second,
```

Comment:
these look different than the defaults for the minion. why? the cadvisor default for `housekeeping_interval` is `10s` -- any reason to use something different here? if so, can you please comment as to why?